test/EndToEnd/tests/NetCoreProjectTest.ps1

```diff
@@ -163,6 +165,8 @@ function Test-NetCoreVSandMSBuildNoOp {
 }
 
 function Test-NetCoreTargetFrameworksVSandMSBuildNoOp {
+    [SkipTest('https://github.com/NuGet/Home/issues/13003')]
```

Comment:
I updated the PR description, but this is failing because I forgot to update dotnet/project-system to pass `NuGetAuditMode` in nomination:

* https://github.com/NuGet/Home/issues/13003

We can either:

1. Merge this PR with the skip and then I'll need to undo the skip once project-system merged my PR and inserts into VS
2. Wait until project-system merges and inserts the fix, then I need to update this PR to remove these skips.

Reply:
personally, I'm fine with merging the skip.